README.md

```diff
@@ -22,9 +22,46 @@ Top 5 accuracy: 0.89814
 
 # openvino_quantization
 
+Use this script before converting:
+``bash
+import tensorflow as tf
+import argparse
+from tensorflow.python.tools import optimize_for_inference_lib
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--graph', help='.pb graph path', default='resnet_v2_101_299_frozen.pb')
+argv = parser.parse_args()
+
+pb_file = argv.graph
+graph_def = tf.compat.v1.GraphDef()
+
+try:
+    with tf.io.gfile.GFile(pb_file, 'rb') as f:
+        graph_def.ParseFromString(f.read())
+except:
+    with tf.gfile.FastGFile(pb_file, 'rb') as f:
+        graph_def.ParseFromString(f.read())
+
+graph_def = optimize_for_inference_lib.optimize_for_inference(graph_def, ['input'], ['output'], tf.float32.as_datatype_enum)
+
+with tf.gfile.FastGFile('resnet_v2_101_299_opt.pb', 'wb') as f:
+   f.write(graph_def.SerializeToString())t
+``
```

Comment:
Please check rendered README.md - it's unreadable https://github.com/likholat/openVINO_quantization/blob/openvino_test/README.md

Reply:
It's even better to move this script to root folder